### ISSUE TITLE
Hero image not appearing in Android 6.0

### ISSUE THREAD

**Issue by willemruys** [mjmlio/mjml#2067]
**Describe the bug**
Hero image not appearing in Android 6.0

**To Reproduce**

1. My mjml file
I am creating an hero class using mj-class and rendering it in my body using mj-hero:

```

<mjml>
  <mj-head>
    <mj-attributes>
      <mj-class name="title" font-size="30px" font-weight="1000" />
      <mj-class name="subtitle" font-size="22px" font-weight="1000" />
      <mj-class name="date-field" font-size="15px" font-style="italic" />
      <mj-class name="link" color="black" />
      <mj-class
        name="footer-link"
        background-color="transparent"
        color="black"
        font-size="10px"
      />
      <mj-class
        name="button-text"
        background-color="transparent"
        color="black"
        font-size="10px"
        text-decoration="underline"
        text-align="left"
      />
      <mj-class
        name="hero-background-image"
        background-url="https://cloud.githubusercontent.com/assets/1830348/15354890/1442159a-1cf0-11e6-92b1-b861dadf1750.jpg"
        background-size="cover"
        background-repeat="no-repeat"
        background-color="white"
        border-bottom="solid 5px #22B14C"
        padding="20px"
        mode="fixed-height"
        background-height="120px"
        height="120px"
        background-width="600px"
      />
    </mj-attributes>
    <mj-style inline="inline">
      .card { background-color: grey; height: 200px; }
    </mj-style>
  </mj-head>
  <mj-body background-color="#e6e6e6">
    <mj-raw>
      <!-- navbar -->
    </mj-raw>
    <mj-section full-width="full-width" padding="0px">
      <mj-column background-color="white">
        <mj-image
          src="https://www.dutchdatacenters.nl/wp-content/uploads/2019/06/ABB-1.png"
          alt="ABB"
          height="50px"
          width="100px"
          align="right"
        />
      </mj-column>
    </mj-section>
    <mj-raw>
      <!-- hero -->
    </mj-raw>
    <mj-hero mj-class="hero-background-image">
      <mj-text color="white"> Welcome to <strong>MY EMAIL</strong></mj-text>
    </mj-hero>
...
 </mj-body>
</mjml>
```
2. How i export it to html
I am using the VScode plugin export html function

3. Previewing email in clients
I paste the html to litmus builder
Android 6.0 is not showing background.

**Expected behavior**

**MJML environment (please complete the following information):**
 - OS: [e.g. MacOS]
 - MJML Version [e.g. 4.2.0]
 - MJML tool used [e.g MJML App]

**Email sending environment(for rendering issues)**:
 - I am previewing the emails in litmus

**Affected email clients (for rendering issues):**
 - Android 6.0

**Screenshots**
<img width="749" alt="Screenshot 2020-10-27 at 11 08 02 AM" src="https://user-images.githubusercontent.com/49044305/97287306-b6e16680-1844-11eb-85f3-51ea189253af.png">
<img width="745" alt="Screenshot 2020-10-27 at 11 08 54 AM" src="https://user-images.githubusercontent.com/49044305/97287440-e1332400-1844-11eb-9fb8-bc6e40960642.png">

**Comment by iRyusa** [comment]
Can you try with an mj-section with background image if it does the same ? Default Android client isn't really used anymore but idk if it did support background image at some point 🤔

**Comment by Pharmasolin** [comment]
I can confirm this bug (saw on litmus), I didn't dive into this issue because I didn't need to target Android 5.1, 6 email clients, but I thinks that's related to Yahoo issue with hero image, with url inside `background` html attribute (but I didn't test it, just a thought)

**Comment by Pharmasolin** [comment]
I dived into this bug while working on #2122 and I tried different variations of background image setting but none worked, after some research I made a conclusion that mjml-hero might work in Android 6.0 (default email client, not gmail) but it seems like litmus didn't update their app.

https://blog.edmdesigner.com/background-images-in-modern-html-emails/#:~:text=updates%20were%20not%20pushed%20on%20Litmus

And it looks like those email clients are not supported anymore and users on those devices should use gmail app (where background image works, even in gmail imap).

So this one can be closed, nothing can be done with it.
